container/lxd/lxd.go

```diff
@@ -140,6 +157,8 @@ func (manager *containerManager) CreateContainer(
 		"boot.autostart": "true",
 	}
 
+	ApplyConstraints(metadata, cons)
```

Comment:
As noted in another comment; applying is one thing, validating is another. Can we validate? Should LXD validate, etc?